lib/constants.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  KEY_START: '-----BEGIN PRIVATE KEY-----',
+  KEY_END: '-----END PRIVATE KEY-----',
+  RSA_KEY_START: '-----BEGIN RSA PRIVATE KEY-----',
+  RSA_KEY_END: '-----END RSA PRIVATE KEY-----',
+  CERT_START: '-----BEGIN CERTIFICATE-----',
+  CERT_END: '-----END CERTIFICATE-----'
```

Comment:
you do any research on these? I'm not 100% sure these are the only ones we would want to accept  - and if there are more, probably want to refactor this to be more just `PRIVATE KEY`, `CERTIFICATE`, etc.., and then you can generate the matches using `-----BEGIN ${theType}-----` and `-----END ${theType}-----` kind of stuff. or maybe regular expressions.